bridge_adaptivity/module/policies/base.py

```diff
@@ -27,6 +27,8 @@ class BaseGradingPolicy(object):
 
     public_name = 'Grading Policy'
     require = {}
+    summary_text = ''
```

Comment:
Some comments about these attributes should be used in subclasses might be helpful, either inline or in this class docstring.

Reply:
ok, will add